frontend/components/landing/LandingTechBanner.vue

```diff
@@ -5,17 +5,17 @@
     <h2
       class="w-10/12 text-2xl font-semibold text-center sm:text-3xl sm:w-full xl:text-4xl"
     >
-      {{ $t("landing-page-github-header") }}
+      {{ $t("components.landing.tech-banner.github-header") }}
```

Comment:
Similarly here we'd have `components.landing.landing-tech-banner.github-header` :)

Reply:
I'll fix that